src/main/scala/Dispatcher.scala

```diff
@@ -0,0 +1,41 @@
+import scala.io.Source
+
+import telegram.API._
+import tinkoff.API._
+
+/**
+  * Main polling loop.
+  */
+object Dispatcher extends App {
+
+  def getFormattedRates(rates: List[Rate]): String = {
+    val textRatesList = rates.map(x => "%3s: %7.3f  %7.3f".format(x.fromCurrency.name, x.buy, x.sell.get))
+    "<pre>" +
+      "       Buy      Sell\n" +
+      "      ------   ------\n" +
+      textRatesList.mkString("\n") + "</pre>"
+  }
+
+  setToken(Source.fromFile("TelegramBotToken").getLines.mkString)
+
+  while (true) {
+    val updatesList = getUpdates
+
+    for (update <- updatesList; if update.message.isDefined; message = update.message.get) {
```

Comment:
Message is not iterable. If you meant to make a list of messages and iterate over it, 'for' seems more appropriate. In future we probably will use other fields of update